modules/32_power/DTcoup/equations.gms

```diff
@@ -73,17 +73,10 @@ q32_limitCapTeStor(t,regi,teStor)$( t.val ge 2015 ) ..
 *** with VRE capacities which require storage (according to q32_limitCapTeStor): 
 
 
-*** build additional electrolysis capacities with stored VRE electricity
-q32_elh2VREcapfromTestor(t,regi)..
-  vm_cap(t,regi,"elh2","1") 
-  =g= 
-  sum(te$testor(te), p32_storageCap(te,"elh2VREcapratio") * vm_cap(t,regi,te,"1") )
-;
-
 *** build additional h2 to seel capacities to use stored hydrogen
 q32_h2turbVREcapfromTestor(t,regi)..
   vm_cap(t,regi,"h2turbVRE","1") 
-  =e= 
+  =g= 
   sum(te$testor(te), p32_storageCap(te,"h2turbVREcapratio") * vm_cap(t,regi,te,"1") )
```

Comment:
I think we don't need this greater than change here becaues we didn't add ngt here yet.

Reply:
@fschreyer : I agree, this doesn't need to be changed unless you also add ngt here. 
@cchrisgong : I am surprised that the H2 requirement is still on in the Dieter-coupled version - is this on purpose? 
or is it simply not binding, because pv and wind are not defined as testor anymore?
in the main model version, we are now allowing also ngt to serve as backup - I guess you would want the same in the DIETER-coupled version?

Reply:
Changed it back (in both realizations, `DTcoup` and `IntC`)